packages/lumx-react/src/components/table/TableRow.tsx

```diff
@@ -64,11 +69,21 @@ const TableRow: React.FC<TableRowProps> = ({
     children,
     className = '',
     isClickable = DEFAULT_PROPS.isClickable,
+    isDisabled = DEFAULT_PROPS.isDisabled,
     isSelected = DEFAULT_PROPS.isSelected,
     ...props
 }: TableRowProps): ReactElement => (
     <tr
-        className={classNames(className, handleBasicClasses({ prefix: CLASSNAME, isClickable, isSelected }))}
+        className={classNames(
+            className,
+            handleBasicClasses({
+                isClickable: isClickable && !isDisabled,
+                isDisabled,
+                isSelected: isSelected && !isDisabled,
```

Comment:
What if we want to have a selected disable row? We could have a not-unselectable row :man_shrugging: